src/cdp/router.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { StubServer } from '../tests/stubServer.spec';
+
+import * as chai from 'chai';
+import chaiAsPromised from 'chai-as-promised';
+chai.use(chaiAsPromised);
+
+import * as sinon from 'sinon';
+
+import { MessageRouter } from './router';
+
+describe('MessageRouter', function () {
+  it('can send a message without a sessionId', async function () {
+    const mockCdpServer = new StubServer();
+    const router = new MessageRouter(mockCdpServer);
+
+    const message = JSON.stringify({ id: 0, method: 'Browser.getVersion' });
+    router.sendMessage(message);
+
+    sinon.assert.calledOnce(mockCdpServer.sendMessage);
+    sinon.assert.calledWith(mockCdpServer.sendMessage, message);
+  });
+
+  it('can send a message with a sessionId', async function () {
+    const mockCdpServer = new StubServer();
+    const router = new MessageRouter(mockCdpServer);
+
+    const message = JSON.stringify({
+      sessionId: 'ABCD',
+      id: 0,
+      method: 'Page.enable',
+    });
+    router.sendMessage(message);
+
+    sinon.assert.calledOnce(mockCdpServer.sendMessage);
+    sinon.assert.calledWith(mockCdpServer.sendMessage, message);
+  });
+
+  it('routes event messages to the correct handler based on sessionId', async function () {
+    const mockCdpServer = new StubServer();
+    const router = new MessageRouter(mockCdpServer);
+
+    const browserMessage = { method: 'Target.attachedToTarget' };
+    const sessionAMessage = { sessionId: 'A', method: 'Page.frameNavigated' };
```

Comment:
Small suggestion: What about more readable values like `SESSION_ID_1`, or `SOME_SESSION_ID` and `ANOTHER_SESSION_ID`?